Makefile

```diff
@@ -175,70 +167,123 @@ tidy: clean
 	GO111MODULE=on go mod tidy
 	git diff -U --exit-code go.mod go.sum
 
-update-install-scipt:
+install.sh:
 	./hack/update_install_script.sh
 
-check-install-script: update-install-scipt
+check-install-script: install.sh
 	git diff -U --exit-code install.sh
 
 clean:
-	rm -rf docs/docs.go
+	rm -rf docs/docs.go $(CLEAN_TARGETS)
 
 boilerplate:
 	./hack/verify-boilerplate.sh
 
-taily-build:
-	if [ "$(shell docker ps --filter=name=$@ -q)" = "" ]; then \
-		docker build -t pingcap/chaos-binary ${DOCKER_BUILD_ARGS} .; \
-		docker run --rm --mount type=bind,source=$(shell pwd),target=/src \
-			--name $@ -d pingcap/chaos-binary tail -f /dev/null; \
-	fi;
-
-taily-build-clean:
-	docker kill taily-build && docker rm taily-build || exit 0
+image: image-chaos-daemon image-chaos-mesh image-chaos-dashboard
```

Comment:
I run `make image` on my local computer, but get the error below:
```log
make: *** No rule to make target `image-chaos-daemon', needed by `image'.  Stop.
```

Reply:
Remove `=` will solve this problem. Fixed